player.go

```diff
@@ -299,3 +299,12 @@ func (p *Player) setMedia(m *Media) error {
 	C.libvlc_media_player_set_media(p.player, m.media)
 	return getError()
 }
+
+// SetXWindow sets the X window to play on
```

Comment:
Can you please add a "." at the end of the comment sentence?
```
// SetXWindow sets the X window to play on.
```

Reply:
Done